src/views/RecipeItem.vue

```diff
@@ -4,11 +4,10 @@
       <img :src="recipe.url" :alt="recipe.title" />
     </div>
     <div class="fullsize-recipe-container-details">
-      <a href="#">
-        <router-link :to="{ name: 'Home' }"
-          ><i v-on:click="closeRecipe" class="fas fa-times"></i
-        ></router-link>
-      </a>
+      <router-link :to="{ name: 'Home' }"
+        ><i v-on:click="closeRecipe" class="fas fa-times"></i
```

Comment:
You need to remove `v-on:click="closeRecipe"` since it's not more needed

Reply:
@probil Done!